scripts/zones/Arrapago_Reef/mobs/Lamia_Exon.lua

```diff
@@ -0,0 +1,31 @@
+-----------------------------------
+-- Area: Arrapago Reef
+--  Mob: Lamia Exon
+-- Note: When this mob dies, it updates a timer to be respawned
+--       by Medusa.
+-----------------------------------
+mixins = { require("scripts/mixins/weapon_break") }
+-----------------------------------
+local entity = {}
+
+entity.onMobSpawn = function(mob)
+    -- Two of Medusa's adds spawn with different models and jobs
+    if mob:getID() > zones[mob:getZoneID()].mob.MEDUSA + 2 then
+        mob:setMobMod(xi.mobMod.SPELL_LIST, 0)
+        mob:setMagicCastingEnabled(false) -- Remove once SPELL_LIST mobMod is fixed
```

Comment:
Maybe a module because I suspect only you would put a PR in to remove this across the board? Whereas a module might get more oversight?

Reply:
Can we make modules for files that don't exist? :(

Reply:
I see. I thought you meant once it was fixed, implying it was there but borked somehow.

Reply:
ah yeah it was just my fault using it improperly